lesson-13/script.js

```diff
@@ -0,0 +1,181 @@
+window.onload = input;
```

Comment:
Name `input` is not meaningful 

```suggestion
window.onload = input;
```